### PR TITLE
MicrosoftTeamsAsk - added channel argument

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
Added `channel` argument to `MicrosoftTeamsAsk` script

## Required version of Demisto
5.0.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation
- [ ] Code Review

## Dependencies
None

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 please review yml and md

**Comment by kirbles19** [comment]
@Itay4 done

File: Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml [hunk 1]
```diff
@@ -17,40 +17,46 @@ args:
...
   secret: false
-comment: Ask a user a question on Microsoft Teams and expect a response. The response
-  can also close a task (might be conditional) in a playbook.
+- default: false
+  description: Channel to which to send the question.
+  isArray: false
+  name: channel
+  required: false
+  secret: false
+comment: Send a user a question with predefined response options on Microsoft Teams. The response
```
Comment (yaakovi) [inline Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml]: user or channel
also, be consistent: "team member" or "user"

File: Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 args:
 - default: false
-  description: Question message to send to the team member
+  description: Question (message) to send to the specified team member.
```
Comment (yaakovi) [inline Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml]: team member or channel

File: Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml (hunk 1 above)
Comment (kirbles19) [reply Scripts/MicrosoftTeamsAsk/MicrosoftTeamsAsk.yml]: @yaakovi Absolutely right. Let's go with "team member"

Send a team member or channel a question with predefined....

**Pull request merged**